recipes-bsp/uefi/edk2-firmware-tegra/0001-l4tlauncher-disable-a-b-rootfs-validation.patch

```diff
@@ -0,0 +1,40 @@
+From 739813f08471028eae87002e19f55e8b59083fd6 Mon Sep 17 00:00:00 2001
```

Comment:
My understanding of bitbake is that if you skip adding this file, it'll find the one in the original rule by the search path.  That'll keep it in sync better over time, I think.

Happy to merge as is if you want.

Reply:
I think merging as-is is probably easier. I already ran into trouble earlier thinking the patch got built into the firmware when it turned out it wasn't because of messing up the bitbake file